client/src/app/components/managing-body/managing-body.component.html

```diff
@@ -0,0 +1,15 @@
+<div class="managingBody" (click)="onClick()">
+    <div class="">
```

Comment:
it is written like that, cause if we are gonna add semantic-ui the classes will be filled 
with 'ui-grid' on top
and each one with his own line size..